lib/inferno/apps/web/controllers/test_suites/index.rb

```diff
@@ -3,8 +3,8 @@ module Web
     module Controllers
       module TestSuites
         class Index < Controller
-          def call(_params)
-            self.body = serialize(repo.all, view: :summary)
+          def handle(_req, res)
```

Comment:
I think `_req` reads as "a request variable I'm not using" where `_` is "a variable I'm not using".  You could maybe infer it because of familiar and common request/response pair but I think I prefer `_req`.  It's not a rubyism to me.

Reply:
Yeah, just for readability. I think it's always a good idea in method signatures to indicate what the parameter is even if you're not using it. If it's just a block where you're doing something like `some_hash.each { |_, value| ... }`, then I think it's fine to just use `_`.